teamcity/email.go

```diff
@@ -49,14 +50,18 @@ func (r *emailResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 	resp.Schema = schema.Schema{
 		Description: "TeamCity can notify users about various events using email. More info [here](https://www.jetbrains.com/help/teamcity/configuring-notifications.html)",
 		Attributes: map[string]schema.Attribute{
-			"enabled":  schema.BoolAttribute{Required: true},
-			"host":     schema.StringAttribute{Required: true},
-			"port":     schema.Int64Attribute{Required: true},
-			"from":     schema.StringAttribute{Required: true},
-			"login":    schema.StringAttribute{Required: true},
-			"password": schema.StringAttribute{Required: true, Sensitive: true},
+			"enabled": schema.BoolAttribute{Required: true},
+			"host":    schema.StringAttribute{Required: true},
+			"port":    schema.Int64Attribute{Required: true},
+			"from":    schema.StringAttribute{Required: true},
+			"login": schema.StringAttribute{
+				Optional: true,
+				Computed: true,
+				Default:  stringdefault.StaticString(""),
+			},
+			"password": schema.StringAttribute{Optional: true, Sensitive: true},
 			"secure_connection": schema.StringAttribute{
-				Required: true,
+				Optional: true,
```

Comment:
If it becomes optional, then does not it require Default value? the same question about `password` field

Reply:
I checked that for a default (fresh new) TeamCity instance we have this in the response, so password has null as a default value, I think we dont need to set this, but for the secure connection yes, thank you, Ill set the default value to "NONE"
![image](https://github.com/JetBrains/terraform-provider-teamcity/assets/51397216/952145e8-eb89-4550-aa1f-926fca06addb)